### PR TITLE
Load transform mehrdad

### PR DESCRIPTION
**Pull request by mehrdadhalali**

### CODE REVIEW

File: load_to_rds/README.md [hunk 1]
```diff
@@ -26,3 +26,10 @@ DB_NAME=XXXXX
 
 
 ## 📄 Files Explained
+
+- `get_data_from_database.py`: To upload the data, we need to know some things about the already existing data, like IDs and duplicates. This script is for querying that data.
+- `transform_game_data.py`: This is a short script that transforms the data received by the scrapers.
+- `upload_to_db.py`: This script uploads all of the gathered data to the database.
+- `test_load_to_rds`: Testing code.
+- `delete.sql`: An SQL script to delete all game data (NOT metadata), executed by `clear_database.sh`
+- `connect.sh`: Connects to the database.
```
Comment (paulauus) [inline load_to_rds/README.md:35]: More, give me more info. In further prs.

File: load_to_rds/upload_to_db.py [hunk 2]
```diff
@@ -22,7 +29,7 @@ def remove_duplicates(listings_list: list[dict], already_scraped: list[str]) ->
 
 def update_genres(new_genres: list[str], conn: connection) -> dict:
     """Adds any new genres to the genre table, returning their id map."""
-
+    print(new_genres)
```
Comment (paulauus) [inline load_to_rds/upload_to_db.py:32]: Good to check. I always forget to remove these for a while

**Review by paulauus — DISMISSED** [review]

File: load_to_rds/get_data_from_database.py [hunk 3]
```diff
@@ -41,7 +41,7 @@ def get_ids(table_name: str, conn: connection, column_prefix: str = None) -> dic
     return name_to_id
 
 
-def get_listings_for_the_day(day: datetime.date, conn: connection) -> list[str]:
+def get_listings_for_the_day(day: datetime, conn: connection) -> list[str]:
```
Comment (Leon-JS02) [inline load_to_rds/get_data_from_database.py:44]: Good spot

**Review by Leon-JS02 — DISMISSED** [review]
Good changes.

**Review by Leon-JS02 — APPROVED** [review]
Reapproval - removed an unneeded print statement.

**Review by paulauus — APPROVED** [review]

**Pull request merged**